src/helpers/stringHelper.js

```diff
@@ -1,31 +1,30 @@
-import * as R from 'ramda';
+import * as R from "ramda";
 
-export const invalidFirstName = (firstName)=> {
-    return (
-        R.isEmpty(firstName) ||
-        R.gt(firstName.length, 32) ||
-        R.lt(firstName.length, 3)
-    );
+export const invalidFirstName = firstName => {
+  return (
+    R.isEmpty(firstName) ||
+    R.gt(firstName.length, 10) ||
```

Comment:
We're changing char limits for first and last names to 10?

Reply:
to match API

Reply:
Gotcha.